src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -24,36 +23,36 @@ public class QualificationRunner : QualificationRunner<Project, PKSimProject>
       private readonly ICoreWorkspace _workspace;
       private readonly IWorkspacePersistor _workspacePersistor;
       private readonly IExportSimulationRunner _exportSimulationRunner;
-      private readonly IMarkdownReporterTask _markdownReporterTask;
+      private readonly IQualificationInputTask _qualificationInputTask;
 
       public QualificationRunner(ISnapshotTask snapshotTask,
          IJsonSerializer jsonSerializer,
          ICoreWorkspace workspace,
          IWorkspacePersistor workspacePersistor,
          IExportSimulationRunner exportSimulationRunner,
          IDataRepositoryExportTask dataRepositoryExportTask,
-         IMarkdownReporterTask markdownReporterTask,
+         IQualificationInputTask qualificationInputTask,
          IOSPSuiteLogger logger
       ) : base(logger, dataRepositoryExportTask, jsonSerializer, snapshotTask)
       {
          _workspace = workspace;
          _workspacePersistor = workspacePersistor;
          _exportSimulationRunner = exportSimulationRunner;
-         _markdownReporterTask = markdownReporterTask;
+         _qualificationInputTask = qualificationInputTask;
       }
 
       protected override void SaveProjectContext(string projectFile) => _workspacePersistor.SaveSession(_workspace, projectFile);
 
       protected override void LoadProjectContext(PKSimProject project) => _workspace.Project = project;
 
-      protected override void ValidateInputs(Project snapshotProject, QualificationConfiguration configuration)
+      protected override void ValidateInputs(Project snapshotProject, QualificationConfiguration configuration) => 
+         _qualificationInputTask.ValidateInputs(snapshotProject, configuration);
+
+      protected override async Task<(PKSimProject, InputMapping[])> LoadProjectAndExportInputs(QualificationRunOptions runOptions, Project snapshot, QualificationConfiguration config)
```

Comment:
This is what I wanted to comment in the upstream PR.  `Export` feels weird in naming because I am expectign a side effects such as inputs being saved as a file
Also loading a project and returning some inputs are two complete different concerns. Why do we put them in one method? (just curious, no need to change)

Reply:
This is actually the case. The project will be created as before, and the input will be processed. That means markdown will be exported to the file system. 

The reason they are together now is that for MoBi, when we load a project from a snapshot, part of the process is recreating the PK-Sim project to export the PK-Sim module. The inputs specified in the qualification are relative to the PK-Sim project, so we have to process the markdown at that time.

At no point does the PK-Sim project exist beyond the duration of the module export so that's why the steps are now together.